jboss-integration/src/test/java/org/teiid/jboss/rest/TestRestWebArchiveBuilder.java

```diff
@@ -64,10 +71,59 @@ public void testBuildArchive() throws Exception {
 		ArrayList<String> files = new ArrayList<String>();
 		files.add("WEB-INF/web.xml");
 		files.add("WEB-INF/jboss-web.xml");
+		files.add("WEB-INF/jboss-deployment-structure.xml");
 		files.add("WEB-INF/classes/org/teiid/jboss/rest/View.class");
 		files.add("WEB-INF/classes/org/teiid/jboss/rest/TeiidRestApplication.class");
 		files.add("META-INF/MANIFEST.MF");
 		
+		files.add("api-doc.html");
```

Comment:
Also do not include all the required "Static" files one by one to the archive, just add them from pre-packed directory of swagger stuff, that way when we need to update swagger version, it is just updating whatever that new directory